Лабораторная работа 2(Курганов Тихон4931101/30003)

looklikecarti opened the pull request:
![Снимок экрана 2024-10-23 в 03 25 40](https://github.com/user-attachments/assets/e55ab489-7113-4332-9a7f-6e408ad178ef)

looklikecarti committed:
Лабораторная работа 2(Курганов Тихон4931101/30003)

ZebraHead22 reviewed (commented):
18783 конечно правильно...
Но не совсем. Вы просто округляете. 
Но если вы округлите вниз, то денег не хватит...)

looklikecarti committed:
Update task2_lab2.py

добавил модуль math, использовал функцию ceil для строго округления в большую сторону.

looklikecarti commented:
> 18783 конечно правильно... Но не совсем. Вы просто округляете. Но если вы округлите вниз, то денег не хватит...)
Обновил свой код, добавил модуль math, использовал функцию ceil для строго округления в большую сторону.
